viv_utils/flirt.py

```diff
@@ -46,6 +45,40 @@ def is_library_function(vw, va):
     return vw.funcmeta.get(va, {}).get(_LIBRARY_META_KEY, False)
 
 
+def is_library_called_function(vw, va):
```

Comment:
Is this function name descriptive?

Reply:
`is_only_called_from_library_functions`?